server/grpc_service.go

```diff
@@ -678,7 +744,7 @@ func (s *GrpcServer) GetStore(ctx context.Context, request *pdpb.GetStoreRequest
 	} else if rsp != nil {
 		return rsp.(*pdpb.GetStoreResponse), err
 	}
-
+	time.Sleep(120 * time.Millisecond)
```

Comment:
Why sleep?

Reply:
Sorry, I forgot to delete this when addressing the comment. I can't count concurrency without this during manual testing.

Reply:
need remove it.